### ISSUE TITLE
Support request for stream.meet.google.com

### ISSUE THREAD

**Issue by FelixFrog** [ytdl-org/youtube-dl#27736]
## Checklist

- [x] I'm reporting a new site support request
- [x] I've verified that I'm running youtube-dl version **2021.01.08**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that none of provided URLs violate any copyrights
- [x] I've searched the bugtracker for similar site support requests including closed ones

## Example URLs

- Single video: https://stream.meet.google.com/stream/87210a1c-77c2-4026-942b-2d44e4702f2f

## Description

This is a feature of google meet avaliable to G Suite Enterprise for Education, Enterprise Standard and Enterprise Plus users where it is possible to livestream a conference (more info https://support.google.com/meet/answer/9308630). It requires authentication but that works fine just with cookies. It embeds a "www.youtube-nockookie.com" frame, that viewed alone doesn't work and straight up crashes.

**Comment by JustAnotherFormerLurker** [comment]
I think this is a bad idea. Google Meet for Education (and many other service offerings) already allows for recording the video stream (https://support.google.com/meet/answer/9308681). But this is only available to teachers due to problems with cyber bullying at schools.

If the conference is to be taped for later use, the people administering it can do it. 

I think youtube-dl is a great way to archive videos that might otherwise be lost, but in this case I think it would be opening pandoras box. Google Meet is one of the solutions many schools have been forced to use in this pandemic and the teachers and staff are overworked and underequipped to handle many issues that arise. Right now many schools are battling the problems that have risen when kids that don't understand better record or take screenshots and use them to hurt other students, and the schools cannot police this quickly enough.

If you have a legitimate use case for this feature, please present it. But as recording is already easily available to those that organize the meeting, I would rather see the youtube-dl team focus on other matters.

**Comment by FelixFrog** [comment]
I totally agree with you. Yet, the use case I am proposing here is the ability to get the stream url/playlist file to be able to stream it with other tools and without a browser (for example mpv) since the built-in interface has no options whatsoever. 

> Google Meet for Education (and many other service offerings) already allows for recording the video stream (https://support.google.com/meet/answer/9308681).

This a different feature. What the document above refers to is the possiblity of recording Google meet rooms from the inside as a member, and is a commonly used features for teachers or small private meetings. The links like the ones described above (https://stream.meet.google.com/stream/87210a1c-77c2-4026-942b-2d44e4702f2f) are usually given to a broad audience, in order to make people watch a meeting even when the room is full (more than 250 people), and are therefore of public domain inside the organisation. They are essentially a livestream, where the viewers cannot interact at all. Thus, this does not encourage any form of bullying/mocking in private environments and, in fact, helps the preservation of livestreams for academic purposes.
